### ISSUE TITLE
Metamask/other standard block explorer link generation doesn't work

### ISSUE THREAD

**Issue by lovelaced** [paritytech/contract-issues#5]
When a tx hash is emitted, links are created in the format $BLOCK_EXPLORER/tx/$HASH. We should ensure subscan supports this so we can work with it out of the box pending Etherscan integration.

**Comment by pgherveou** [comment]
Yes we need to build some integration for supporting EVM style transaction in block explorer, and get this to work.
This resolve around the fact that subscan identify transaction with `<blockhash>/<tx index>` while Ethereum transaction are simply indexed by the transaction hash

**Comment by pgherveou** [comment]
Marking as closed as we do that already
we have a small proxy that redirect to the correct subscan url
You can set https://westend-asset-hub-eth-explorer.parity.io as the block explorer in MetaMask

code for the proxy script is here https://github.com/pgherveou/ah-westend-subscan
